cl/alerts/management/commands/cl_send_recap_alerts.py

```diff
@@ -98,7 +98,7 @@ def compute_estimated_remaining_time(
         initial_wait,
     )
 
-    return estimated_time_remaining
+    return min(estimated_time_remaining, 7200)
```

Comment:
Would it be possible to return the original estimated time here and use `min(estimated_time_remaining, 7200)` only within the sleep call inside the while loop? This way, we can still know the actual estimated time while checking the task status more frequently. Additionally, I think it would be more informative if we logged the estimated time more frequently, perhaps every 15 minutes (900 seconds) instead of every 2 hours (7200)?

Reply:
> only within the sleep call inside the while loop? 

Sure!

> perhaps every 15 minutes (900 seconds) instead of every 2 hours (7200)?

Alright! I'll apply your suggestion.